### PR TITLE
First attempt on supporting launches jar from file or GAV.

### PR DESCRIPTION
**Pull request by maxandersen**
Basically just says "if script ends in jar run with jar" and
when resolving sarguments GAV's gets honored

### CODE REVIEW

**Comment by maxandersen** [comment]
relates to #45

File: src/main/java/dev/jbang/DependencyUtil.java [hunk 1]
```diff
@@ -42,15 +44,23 @@
...
-	 * 
+	 *
 	 * @param deps
 	 * @param repos
 	 * @param loggingEnabled
 	 * @return string with resolved classpath
 	 */
 	public String resolveDependencies(List<String> deps, List<MavenRepo> repos,
-			boolean offline, boolean loggingEnabled) {
+			boolean offline, boolean loggingEnabled, boolean transivity) {
```
Comment (quintesse) [inline src/main/java/dev/jbang/DependencyUtil.java:63]: I think it's "transitivity" :-)

Comment (maxandersen) [reply src/main/java/dev/jbang/DependencyUtil.java]: fixed

**Comment by maxandersen** [comment]
the changes wasn't that big and it doesn't break existing functionallity thus merged.

**Comment by quintesse** [comment]
Yay!

**Pull request merged**